### PR TITLE
All the modified flowtools have been tested to work with Galaxy 19.01

### PR DESCRIPTION
**Pull request by zph001**
All the modified **flowtools** have been tested to work with Galaxy 19.01 and also work with docker-galaxy-stable/compose **containers** 19.01

### CODE REVIEW

File: tools/flowtools/analysis/metacyto_autocluster.xml.orig [hunk 1]
```diff
@@ -0,0 +1,137 @@
+<tool id="metacyto_autocluster" name="Autoclustering analysis" version="1.0">
```
Comment (bgruening) [inline tools/flowtools/analysis/metacyto_autocluster.xml.orig:1]: remove this one ...

File: tools/flowtools/fcs_tools/FCS1Dplotggcyto.xml.orig [hunk 2]
```diff
@@ -0,0 +1,117 @@
+<tool id="ggcyto_1d_density_plots" name="Generate 1D density plots" version="1.0">
```
Comment (bgruening) [inline tools/flowtools/fcs_tools/FCS1Dplotggcyto.xml.orig:1]: please remove this as well ..

**Review by bgruening — COMMENTED** [review]
@zph001 I think you should remove the orig files :)

File: tools/flowtools/analysis/metacyto_autocluster.xml.orig (hunk 1 above)
Comment (zph001) [reply tools/flowtools/analysis/metacyto_autocluster.xml.orig:1]: Oh yes, thanks

**Comment by dunnpa** [comment]
Hi zph001 and bgruening,

The ImmPort Galaxy team is wondering if the changes being made are part of a Galaxy-wide effort to upgrade instances to Galaxy 19.01 or if you have a more specific interest in mind.

**Comment by zph001** [comment]
I'll create a new pull and remove all *.orig

File: tools/flowtools/analysis/metacyto_autocluster.xml.orig (hunk 1 above)
Comment (zph001) [reply tools/flowtools/analysis/metacyto_autocluster.xml.orig:1]: All *.orig are removed in a new pull.

**Comment by zph001** [comment]
> Hi zph001 and bgruening,
> 
> The ImmPort Galaxy team is wondering if the changes being made are part of a Galaxy-wide effort to upgrade instances to Galaxy 19.01 or if you have a more specific interest in mind.

Hi the ImmPort Galaxy team, Thanks for your good work on flowtools. We have made some modifications to these tools (*.xml files) and they now work with our galaxy 19.01 instance.

**Comment by bgruening** [comment]
> The ImmPort Galaxy team is wondering if the changes being made are part of a Galaxy-wide effort to upgrade instances to Galaxy 19.01 or if you have a more specific interest in mind.

There is no such effort, but we recommend always to use the latest version :). I think what happens here, correct me if I'm wrong @zph001, is that people install your great tools in different Galaxy instances and are eager to help to keep the tools up to date.

**Comment by dunnpa** [comment]
@zph001 , 
The ImmPort team would be very interested in chatting with you about your experience using flow tools in Galaxy.   

Cheers,
Patrick

**Comment by zph001** [comment]
> @zph001 ,
> The ImmPort team would be very interested in chatting with you about your experience using flow tools in Galaxy.
> 
> Cheers,
> Patrick

Yes @bgruening and that's the purpose of my fork and PR. Hi Patrick, When cloning **ImmPortDB/immport-galaxy**, some flowtools have failed dependencies because they are outdated or conflict with others. Hence we have to make some modifications.

**Pull request closed without merging**